packages/loader/container-loader/src/container.ts

```diff
@@ -1619,11 +1619,13 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             if (errorMsg !== undefined) {
                 const error = new GenericError(
                     errorMsg,
+                    undefined,
```

Comment:
nit: Add a msg what this is: something like `undefined /* error */` so looking at the call we can tell what we are passing as undefined.

Reply:
```suggestion
                    undefined /* error */,
```